content/java/concepts/array-list/terms/replaceAll/replaceAll.md

```diff
@@ -0,0 +1,56 @@
+---
+Title: '.replaceAll()'
+Description: 'Replaces all occurrences of a given value in an ArrayList.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.replaceAll()`** method of the `ArrayList` class takes an Operator as argument and applies it to every element in the List
+
+## Syntax
+
+Applies `operator` to each element in ArrayList `myArrayList` and replaces the element with a new value
+```pseudo
+myArrayList.replaceAll(operator);
+```
+
```

Comment:
Please describe the parameters in the following format:

- `arrayList`: parameter description
- `UnaryOperator<E> operator`: parameter description